### PR TITLE
DELTASPIKE-1091 add default values to Weld profiles.

### PR DESCRIPTION
**Pull request by manovotn**

### CODE REVIEW

**Comment by manovotn** [comment]
To sum this up:
* Removes the need to use system property to trigger Weld profiles 
* Removes the common profile for all Weld profiles, deps/setups were copied over to relevant profiles (turns out to be the smoothest solution)
* Introduces default values for Weld profiles
* So a simple `mvn clean verify -PWeld${X}` (where X = 1 || 2 || 3) will work
* You can still specify Weld version with `-Dweld.version=...`; obviously the major release version has to match the chosen profile

File: deltaspike/parent/pom.xml [hunk 1]
```diff
@@ -49,7 +49,8 @@
     <properties>
         <java.version>1.6</java.version>
         <owb.version>1.2.7</owb.version>
-        <weld.version>1.1.28.Final</weld.version>
+        <!-- Weld profiles (in parent/code/pom.xml) override this version -->
+        <weld.version>1.1.9.Final</weld.version>
```
Comment (johnament) [inline deltaspike/parent/pom.xml]: why downgrade?

File: deltaspike/parent/code/pom.xml [hunk 2]
```diff
@@ -41,7 +41,7 @@
         <wildfly.arquillian.version>1.0.2.Final</wildfly.arquillian.version>
         <glassfish3.version>3.1.2.2</glassfish3.version>
         <glassfish4.version>4.0</glassfish4.version>
-        <wls.version>12.1</wls.version>
+        <wls.version>12.1</wls.version>
```
Comment (johnament) [inline deltaspike/parent/code/pom.xml]: please clean up the whitespace

File: deltaspike/parent/pom.xml (hunk 1 above)
Comment (manovotn) [inline deltaspike/parent/pom.xml]: This is based on a Gerhard's comment. He claimed that the idea is to list the minimal supported version. Although I strongly dislike dislike this approach, I put it here.
However if you look into Weld1 profile, you can see it is overriden by current version (1.1.33.Final). So this version declaration is naught but a mere information :)

**Pull request merged**